app/views/posts/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Posts by user</h1>
\ No newline at end of file
+<main class="posts-page">
+  <div class="row">
+   <%= render partial: "shared/user_card", locals: {u: @user}%>
+
+     <%@posts.each do |p|%>
+      <%= render partial: "shared/post_card", locals: {p:p}%>
+      <div class="comments-wrapper">
+      <%= render partial: "shared/comments"%>
+      </div>
+     <%end%>
```

Comment:
- [ ] Kindly make sure to use methods that you have created in the models, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.) requirement. You can use the `five_most_recent_comments` method here to show the most recent comments instead of all of them.

- [ ] Kindly make sure to show comments that belong to the selected post and not all of the comments.